docs/klaytn-contracts/dao-contracts/introduction.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: dao contracts
+title: DAO contracts
 sidebar_label: DAO Contracts Introduction
```

Comment:
Can we maintain the convention for contracts like "Introduction" for the side label?

Reply:
sure sure

Reply:
cool

Reply:
I checked for Klaytn contracts, since we have different section of contracts we are  having the title as
```
---
title: ⭐ KIP7
sidebar_label: Introduction
---
```
so for dao will do as 
```
---
title: ⭐ DAO Contracts
sidebar_label: Introduction
---
```